string command
int32 distance
int32 angle
int32 map_or_bot
int32 quatX
int32 quatY
int32 quatZ
int32 quatW
